src/main/java/dev/cdevents/models/PipelineRunFinishedSubject.java

```diff
@@ -105,5 +110,44 @@ public String getErrors() {
         public void setErrors(String errors) {
             this.errors = errors;
         }
+
+        /**
+         * @param o
+         * @return true or false
+         */
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) {
+                return true;
+            }
+            if (!(o instanceof PipelineRunFinishedSubjectContent)) {
+                return false;
+            }
+
+            PipelineRunFinishedSubjectContent that = (PipelineRunFinishedSubjectContent) o;
+
+            if (getPipelineName() != null ? !getPipelineName().equals(that.getPipelineName()) : that.getPipelineName() != null) {
```

Comment:
NIT: Since pipeline name is a `string`, this could be `if (getPipelineName() == that.getPipelineName())` which would return `true` also in case they're both `null`

Reply:
Now updated the test validation by comparing the JsonNodes, `equals` method is no longer needed.